o2a/converter/oozie_converter.py

```diff
@@ -94,14 +95,20 @@ def __init__(
             if output_dag_name
             else os.path.join(output_directory_path, self.dag_name) + ".py"
         )
-        params = {"user.name": user or os.environ["USER"]}
-        params = self.add_properties_to_params(params)
-        params = el_utils.parse_els(self.configuration_properties_file, params)
-        self.params = params
+        self.job_properties = {"user.name": user or os.environ["USER"]}
+        self.configuration_properties: Dict[str, str] = {}
+        self.property_set = PropertySet(
+            job_properties=self.job_properties,
+            configuration_properties=self.configuration_properties,
+            action_node_properties={},
+        )
+        self.update_job_properties()
+        self.read_configuration_properties()
         self.parser = parser.OozieParser(
             input_directory_path=input_directory_path,
             output_directory_path=output_directory_path,
-            params=params,
+            job_properties=self.job_properties,
```

Comment:
Why did not you pass the PropertySet object here?

Reply:
Good point. Will do.

Reply:
Also I will do it in ActionMapper and BaseMapper!

Reply:
It will simplify the code quite a lot if I do it this way it seems.

Reply:
Fixed in e0ff6f1bf016cd3a42d7e3f57dedd40aade8f748